README.md

```diff
@@ -1 +1,38 @@
-# web-pack
\ No newline at end of file
+# web-pack
+In the webpack technology is used to arrange the code.
+
+## Built With
+
+- HTML5, CSS3, JS
+- ES6
+- Webpack
+
+## Live Project
+
+[Click here to view](not available)
+
+## Getting Started
+
+Fork / Clone the repository to use it on your local system.
+
+## Usage
+
+View the file through the source code.
+
+## Authors
```

Comment:
- [ ] OPTIONAL
Kindly change the header title _Authors_ to _Author_, as this project is done solely. This will make your ReadMe appear more professional.